server/src/test/resources/csars/yaml/valid/lamp-csar/scripts/start_myphpapp.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
A php app is no script. I would rename it to `myphpapp.php`. And move it to another folder.